first.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>HTML page with links</title>
+<eta harset="utf-8">
+  <meta name="viewport cotent="width=device-width, initial-scale1">  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jqery.minjs">/script>
+      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></scipt>
+      /stye>
+      </head>
+      <body>
+
+<div class="jumbotron text-center">
+  <h1>Introduction to HTML, Javascript & CSS</h1>
+    <p>My first exersice!</p> 
+    </div>
+      
+      <div class="container">
+        <div class="row">
+    <div class="col-sm-4">
+          <h3>Link for local directory</h3>
+        <p>Desktop directory</p>
+      <p><a  target="_blank" href = "file:///C:/Users/453/Desktop"/>Desktop</a></p>
+          </div>
+      <div class="col-sm-4">
+            <h3>Link for text file from my computer</h3>
+          <p>Text file</p>
+        <p><a  target="_blank" href = "file:///C:/Users/453/Desktop/html/htmlinfo.txt">Text file</a></p>
```

Comment:
I don't have these files on my PC and will not be able to see how it behaves.

Reply:
I think I fixed this problem. I also uploaded the local files so they can be accessed on another computer.